pkgs/top-level/all-packages.nix

```
@@ -21721,6 +21721,8 @@ with pkgs;
   prometheus-postgres-exporter = callPackage ../servers/monitoring/prometheus/postgres-exporter.nix { };
   prometheus-process-exporter = callPackage ../servers/monitoring/prometheus/process-exporter.nix { };
   prometheus-pushgateway = callPackage ../servers/monitoring/prometheus/pushgateway.nix { };
+  inherit (callPackage ../servers/monitoring/prometheus/pve-exporter.nix { })
+    prometheus-pve-exporter;
```

I don't think we need that extra file

```suggestion
  prometheus-pve-exporter = python3Packages.toPythonApplication python3Packages.prometheus-pve-exporter;
```